__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const db = require("../db/connection");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => seed(data));
+
+describe("app", () => {
+  describe("GET /api/topics", () => {
+    test("Status: 200, responds with an array of topic objects, each of which should have the properties: slug, description", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body: { topics } }) => {
+          expect(topics).toHaveLength(3);
+          topics.forEach((topic) => {
+            expect(topic).toEqual(
+              expect.objectContaining({
+                description: expect.any(String),
+                slug: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+    test("Status:404, path not found", () => {
+      return request(app)
```

Comment:
This is a good test to have for something like /api/topic/:topic_id however I'd advise testing something like /api/not_an_endpoint to test your invalid path handling :)